include/libzfs.h

```diff
@@ -779,7 +779,9 @@ enum zfs_nicenum_format {
 	ZFS_NICENUM_1024 = 0,
 	ZFS_NICENUM_BYTES = 1,
 	ZFS_NICENUM_TIME = 2,
-	ZFS_NICENUM_RAW = 3
+	ZFS_NICENUM_RAW = 3,
+	/* RAWTIME is the same as RAW except we print dashes ('-') for zero */
```

Comment:
I'm all for comments but I don't think this one adds much and it breaks up the flow of the enum.  I'd drop it in favor of a more verbose comment above the enum which describes all of the types.